auth-web/src/components/auth/staff/continuation-application/ContinuationApplicationTable.vue

```diff
@@ -45,63 +36,17 @@
             >
               <!-- Displaying Status -->
               <template #[`item.status`]="{ item }">
-                <div>{{ displayStatus(item.status) }}</div>
+                <div>
+                  <span :class="['status-dot', item.status]" />
```

Comment:
I believe the syntax is to use curly, no square, braces.

Reply:
Actually, let me look closer at what you've done here. Might be correct :)

Reply:
Yeah, nice job with this class binding 👍 Looks good. Resolving.

PS Another way to do this is to have both `class` and `class:` props, but what you've done is cleaner!